analyses/snv-callers/README.md

```diff
@@ -0,0 +1,211 @@
+* `results/<caller_name>/<caller_name>_ws_report.html`
+
+### Base change analysis
+
+To evaluate base changes I summarized standard MAF fields as two new variables:
+The `base_change` variable that indicates the exact change in bases from
+concatenating `Reference_Allele`, `>`, and `Allele`.
+The `change` variable is made from the `base_change` variable but groups
+together deletions, insertions, and long (more than a SNV) as their own groups.
+
+*Output for this analysis*
+* `results/<caller_name>/<caller_name>_vaf.tsv`
+* `plots/<caller_name>/<caller_name>_<strategy>_base_change.png`
+
+### Variant Allele Fraction Calculation
+
+Calculate variant allele fraction (VAF) for each variant.
+```
+vaf = (t_alt_count) / (t_ref_count + t_alt_count)
+```
+ This is following the [code used in
+`maftools`](https://github.com/PoisonAlien/maftools/blob/1d0270e35c2e0f49309eba08b62343ac0db10560/R/plot_vaf.R#L39).
+The VAF calculations and other special variables are added to the MAF fields
+ and written to a TSV ending in `_vaf.tsv` in the caller's results folder.
+
+ *Output for this analysiss*
+ * `results/<caller_name>/<caller_name>_vaf.tsv`
+ * `plots/<caller_name>/<caller_name>_<strategy>_depth_vs_vaf.png`
+
+### Genomic Regional Analyses
+
+To analyze what genomic regions the variants are from, I used [Annotatr
+package](https://bioconductor.org/packages/release/bioc/vignettes/annotatr/inst/doc/annotatr-vignette.html) to obtain hg38 genome annotations. This Annotatr object is stored as an RDS file: `hg38_genomic_region_annotations.rds` in the `scratch` directory.
+Mutations are assigned all annotations that they overlap (using  
+  `GenomicRanges::overlap`).
+
+*Output for this analysis*
+* `results/<caller_name>/<caller_name>_regions.tsv`
+* `plots/<caller_name>/<caller_name>_<strategy>_snv_regions.png`
+
+### Tumor Mutation Burden Calculation
+
+To calculate TMB, the sum of the bases included in the WXS or WGS BED regions
+are used as the denominator, depending on the sample's processing strategy.
+
+```
+TMBwxs = sum(mutation_w-in_bedwxs)/(wxs_genome_size/1000000)
+TMBwgs = sum(all_mutations)/(wgs_genome_size/1000000)
+```
+
+Where genome size is calculated from the respective BED file as:
+```
+genome_size = sum(End_Position - Start_Position)
+```
+
+BED regions for WXS samples can be [found here](https://raw.githubusercontent.com/AstraZeneca-NGS/reference_data/master/hg38/bed/Exome-AZ_V2.bed).
+BED regions used for WGS samples are caller specific are from <unknown as of now>
+The sample-wise TMB calculations written to a TSV ending in `_tmb.tsv` in the
+caller's results folder.
+
+*Output for this analysis*
+* `results/<caller_name>/<caller_name>_tmb.tsv`
+* `plots/<caller_name>/<caller_name>_<strategy>_tmb_plot.png`
+
+### COSMIC Mutation Overlap
+
+The COSMIC mutation data were obtained from https://cancer.sanger.ac.uk/cosmic/download
+*To run this analysis, you need to obtain these data.*
```

Comment:
Should we include instructions for doing this?

Reply:
Yes, probably/definitely. Their website has pretty good documentation. But this could definitely be filled out a bit more. Once the analysis get's nailed down a bit more I may come back and make this more detailed.